packages/studio-base/src/components/MessagePipeline/store.ts

```diff
@@ -206,6 +210,10 @@ export function createMessagePipelineStore({
         // Use a regular fetch for all other protocols
         return await builtinFetch(uri, options);
       },
+      getMetadata() {
```

Comment:
Isn't possible to simply call the method written on line 131 here? Why is this function repeated?